renovate.json

```diff
@@ -1,5 +1,5 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:base"],
-  "enabledManagers": ["docker-compose"]
+  "enabledManagers": ["docker-compose","maven-wrapper","gradle-wrapper"]
```

Comment:
I'm confused, didn't we just say we wouldn't extend these managers? Or is this just a try-out,

Reply:
I think we said that that particular user story was only about pinning those Docker Compose image versions. Extending those managers is something we can step by step I think. And yes, it's also a bit of a test I guess.